### ISSUE TITLE
app crash when call dealloc

### ISSUE THREAD

**Issue by cleexiang** [tapwork/HeapInspector-for-iOS#11]
~~~
- (void)tw_dealloc
{
    recordAndRegisterIfPossible(self,"dealloc");
    [self tw_dealloc];
}
~~~

**Comment by tapwork** [comment]
can you paste me the output of the crash?

**Comment by tapwork** [comment]
looks like https://github.com/tapwork/HeapInspector-for-iOS/issues/12

**Comment by tapwork** [comment]
has been fixed with 0.0.7
